python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -237,6 +244,7 @@ def _update(self):
             self.provider.terminate_nodes(nodes_to_terminate)
             for node in nodes_to_terminate:
                 self.node_tracker.untrack(node)
+                self.prom_metrics.stopped_nodes.inc()
             nodes = self.workers()
 
         to_launch = self.resource_demand_scheduler.get_nodes_to_launch(
```

Comment:
We should also increment `self.prom_metrics.stopped_nodes.inc()` here: 
<img width="617" alt="Screen Shot 2021-05-31 at 3 06 04 PM" src="https://user-images.githubusercontent.com/21353794/120245351-bb7a5d00-c221-11eb-9422-ea38f4705148.png">